### PR TITLE
add rockspec of this lib

### LINKED ISSUE #15: would this lib be deployed with luarocks
Hello:
   it would be more nice, if lua-resty-s3 can be deployed with luarock.

### PR DESCRIPTION
**Pull request by OnlyPiglet**
fix #15

### CODE REVIEW

File: lua-resty-s3-0.1-1.rockspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package = "lua-resty-s3"
+version = "0.1-1"
+source = {
+   url = "git+https://github.com/OnlyPiglet/lua-resty-s3.git"
```
Comment (jie123108) [inline lua-resty-s3-0.1-1.rockspec]: Shouldn't this PR link be linked to my repository?

**Review by jie123108 — COMMENTED** [review]
do I need to re-upload the Luarocks package?

File: lua-resty-s3-0.1-1.rockspec (hunk 1 above)
Comment (OnlyPiglet) [reply lua-resty-s3-0.1-1.rockspec:4]: sorry I forget to change this , i will fix this and upload again

Comment (OnlyPiglet) [reply lua-resty-s3-0.1-1.rockspec]: > Shouldn't this PR link be linked to my repository?

i am sorry for forgeting changing this url ,I will fix this and uplaod again

Comment (OnlyPiglet) [reply lua-resty-s3-0.1-1.rockspec]: > Shouldn't this PR link be linked to my repository?

@jie123108 

hello I have changed the source.url of rockspec and I have upload this package on rock hub, the url is https://luarocks.org/modules/OnlyPiglet/lua-resty-s3.


 you can upload this lib youself with the command of 

`luarocks upload lua-resty-s3-0.1-1.rockspec --api-key=$(LUAROCKAPIKEY) --force`

LUAROCKAPIKEY is the identity of your account of rock hub

**Review by jie123108 — APPROVED** [review]

**Pull request merged**